src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/commonmodels/PersonName.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+public class PersonName {
```

Comment:
Class already in merged code.

Reply:
Yeah I know I added it to a separate branch and gave it to the others so that there wouldn't be merge conflicts